src/main/java/org/folio/list/services/ListService.java

```diff
@@ -297,4 +301,38 @@ private List<String> getFieldsFromEntityType(EntityType entityType) {
       .forEach(col -> fields.add(col.getName()));
     return fields;
   }
+
+  public void updateListVersions(UUID listId, String name, String description) {
+    Optional<ListEntity> optionalListEntity = listRepository.findById(listId);
+
+    if (optionalListEntity.isPresent()) {
+      ListEntity list = optionalListEntity.get();
+
+      // Save the current version to list_versions
+      saveListVersion(list);
+
+      // Update the current version
+      list.setName(name);
+      list.setDescription(description);
+
+      listRepository.save(list);
+    }
```

Comment:
Matter of personal taste, but instead of `if (optional.isPresent()) {`, you could do `optional.ifPresent(list -> {`